test/oauth2_unittest.cc

```diff
@@ -118,6 +126,32 @@ TEST_F(OAuth2Test, GetAuthHeaderValueUsingTokenFromMetadataServerAsFallback) {
   EXPECT_EQ("Bearer the-access-token", auth.GetAuthHeaderValue());
 }
 
+TEST_F(OAuth2Test, GetApiRequestErrorMetric) {
+  testing::FakeServer oauth_server;
+  testing::TemporaryFile credentials_file(
+    std::string(test_info_->name()) + "_creds.json",
+    "{\"client_email\":\"user@example.com\",\"private_key\":\"some_key\"}");
+  Configuration config(std::istringstream(
+      "CredentialsFile: '" + credentials_file.FullPath().native() + "'\n"
+  ));
+  Environment environment(config);
+  OAuth2 auth(environment);
+  SetTokenEndpointForTest(&auth, oauth_server.GetUrl() + "/oauth2/v3/token");
+
+  ::opencensus::stats::View errors_view(
+      ::google::GceApiRequestErrorsCumulative());
+
+  // no record exists before internal function sent request to oauth_server
+  EXPECT_THAT(errors_view.GetData().int_data(),
+              ::testing::UnorderedElementsAre());
+
+  auth.GetAuthHeaderValue();
+  ::opencensus::stats::testing::TestUtils::Flush();
+  EXPECT_THAT(errors_view.GetData().int_data(),
+                ::testing::UnorderedElementsAre(::testing::Pair(
```

Comment:
Let's move `::testing::Pair` to the next line, so that the whole pair is kept together?